ts/Core/Legend/Legend.ts

```diff
@@ -1634,40 +1647,48 @@ class Legend {
                         item.setState();
                     })
                     .on('click', function (event: PointerEvent): void {
-                        const strLegendItemClick = 'legendItemClick',
-                            fnLegendItemClick = function (): void {
-                                if ((item as any).setVisible) {
-                                    (item as any).setVisible();
-                                }
-                                // Reset inactive state
-                                setOtherItemsState(
-                                    item.visible ? 'inactive' : ''
-                                );
-                            };
+                        const defaultItemClick = function (): void {
+                            if ((item as any).setVisible) {
+                                (item as any).setVisible();
+                            }
```

Comment:
It can also be an item as a DataClass legend item, but we don't have good typing for this. Also, the item can be `BubbleLegendItem`, but this item does not contain `setVisible` method. Removing this `any` would need a lot of refactoring, so I left ` as any` here. Do you think this refactor should be done now or we can leave it for this moment?

Reply:
Ping @TorsteinHonsi 🤞

Reply:
Okay, let's just leave it for now